src/site/filters/liquid.js

```diff
@@ -511,7 +511,7 @@ module.exports = function registerFilters() {
   // react component `facility-appointment-wait-times-widget`
   // (line 22 in src/site/facilities/facility_health_service.drupal.liquid)
   liquid.filters.healthServiceApiId = serviceTaxonomy =>
-    serviceTaxonomy?.fieldHealthServiceApiId;
```

Comment:
Why are you removing the optional chaining operator?
If you are having problems running this code, make sure that you upgrade node to 14.15.0

Reply:
My bad, @jhonnyoddball - I thought that was a typo! Just put that operator back, and was able to build with new node version.